engine/class_modules/sc_death_knight.cpp

```diff
@@ -4628,7 +4632,7 @@ struct empower_rune_weapon_buff_t : public buff_t
     cooldown -> duration = 0_ms; // Handled in the action
     set_period( p -> spec.empower_rune_weapon -> effectN( 1 ).period() );
     set_trigger_spell( p -> spec.empower_rune_weapon );
-    set_default_value( p -> spec.empower_rune_weapon -> effectN( 3 ).percent() );
+    set_default_value( p -> spec.empower_rune_weapon -> effectN( 3 ).percent() + ( p -> conduits.accelerated_cold->ok() ? p -> conduits.accelerated_cold.percent() : 0 ));
```

Comment:
Wouldn't conduits.accelerated_cold.percent() return 0 if ->ok() returns 0 anyway?

Reply:
It does, I erred on the side of explictness in this instance, but wasn't entirely happy with it.  Changed as suggested.